ui/component/preorderContent/view.jsx

```diff
@@ -153,7 +132,8 @@ export default function PreorderContent(props: Props) {
       userParams,
       claimId,
       stripeEnvironment,
-      preferredCurrency,
+      'USD', // hardcode to USD for the moment
```

Comment:
Why?

Reply:
I'd wanna revert this and keep preferred...would be confusing to have the setting but not use it here. Even easier that they are 1:1.